NugetPackagerAssistant/NugetPackager.cs

```diff
@@ -71,7 +72,13 @@ private void CopyFileToDirectory(string fileName)
         {
             var targetLocation = string.Concat(_publishDirectory, Path.DirectorySeparatorChar, Path.GetFileName(fileName));
             File.Copy(fileName, targetLocation, true);
-            Console.WriteLine("{0} has been published", Path.GetFileName(fileName));
+            Console.WriteLine("    {0} has been published", Path.GetFileName(fileName));
```

Comment:
Why did you put whitespaces at the begining of the string ? If you wanted to put a tab, you could just add ```\t````